tower-balance/src/lib.rs

```diff
@@ -1,19 +1,11 @@
-#[macro_use]
-extern crate futures;
-#[macro_use]
-extern crate log;
-extern crate indexmap;
-extern crate rand;
-extern crate tokio_timer;
-extern crate tower_discover;
-extern crate tower_service;
-extern crate tower_util;
+use rand;
```

Comment:
same here i dont think we need this use?

Reply:
done.